sdk/include/psdk/windowsx.h

```diff
@@ -285,7 +285,7 @@
 #define GetWindowExStyle(hwnd) ((DWORD)GetWindowLong(hwnd,GWL_EXSTYLE))
```

Comment:
You are not using GetWindowLongPtr for these ones?

Reply:
The style is always 32 bits.